brd-android/registration/src/main/res/layout/fragment_registration_enter_email.xml

```diff
@@ -83,19 +83,34 @@
 
             </com.google.android.material.textfield.TextInputLayout>
 
+            <CheckBox
+                android:id="@+id/cb_promotions"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:checked="true"
+                android:text="@string/Registration.EnterEmail.Promotions"
+                android:textSize="12sp"
+                android:textColor="@color/light_text_02"
```

Comment:
Create a new style for CheckBox components in `fabriik-common` module and add it to `FabriikAppTheme.Fabriik` theme. Also, remove `textSize`, `textColor` and `fontFamily` attributes here.